vue-app/src/components/Protected.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+<div>Protected</div>
+</template>
+
+<script>
+
+export default {
+  name: 'protected',
+}
+</script>
+
+<style>
+</style>
```

Comment:
A thought, maybe this component should know about the authentication? In consideration.